web/src/app/views/about-view/about-view.component.html

```diff
@@ -52,6 +52,43 @@ <h3 class="card-title">Schůzky každý týden</h3>
   </div>
 </section>
 
+<!-- About us -->
+
+<section class="py-4">
+ <div class="container">
+    
+    <div class="row">
+     <div class="col-12 text-center mb-3">
+       <h2>Kdo jsme?</h2>
+     </div>
+   </div>
+   <div class="row align-items-center">
+      <div class="col-12 col-md-7">
+       <h4>Základní informace</h4>
+       <ul class="list-unstyled">
+         <li>Jsme skupina sedmi oddílů s celoroční činností pro kluky i holky od 7 do 15 let. Pro členy pořádáme každý týden v pondělí či ve středu oddílové schůzky, o prázdninách třítýdenní tábor a v průběhu roku široké spektrum akcí, mezi kterými si jistě každý najde, co ho bude bavit.</li>
+       </ul>
+       <h4>Na jaké akce se s námi můžete těšit?</h4>
+       <ul class="list-unstyled">
+         <li>Zaměřujeme se hlavně na vodu pro malé i velké. Pro menší a pro klidnější povahy jsou víkendovky na klasické vodácké řeky, jako je Vltava, Ploučnice či Ohře. Milovníky adrenalinu zas můžeme naučit jezdit na kajaku a vzít je i na divočejší potoky a řeky u nás i v zahraničí, dle jejich schopností. Případně je můžem vzít i třeba do slalomového kanálu v Tróji. Sportovci jistě ocení závody, jako ČPV a Tři jezy, kterých se každoročně účastníme.</li>
+       </ul>
+       <h4>A co když je zima? Nebo co když mě voda tolik nebaví?</h4>
+       <ul class="list-unstyled">
+         <li>Nevadí! Ne všechny naše akce jsou na vodu. V našem programu se najdou pěší výlety, příběhové akce v naší klubovně, akce na kolo a v zimě i lyže a běžky. A pro ty, co se kamarádí s vodou, ale ne se zimou, jsou tu bazény, kde si mohou zaplavat, zapádlovat, a třeba se i naučit eskymovat, přestože venku mrzne až praští.</li>

```

Comment:
> Nevadí! Ne všechny **naše** akce jsou na vodu. V **našem** programu se najdou pěší výlety, příběhové akce v **naší** klubovně, akce na kolo a v zimě i lyže a běžky.

Na můj vkus trochu moc _našeho_. Taky příběhová akce v klubovně na mě působí jako nějaká únikovka. Co třeba tohle (tučně změny) ?

> Ne všechny naše akce jsou na vodu. **V programu máme** pěší výlety, příběhové akce **po Praze a okolí**, akce na kolo a v zimě i lyže a běžky.

Reply:
A už končím, je to naprosto skvělé Grejdy, moc děkuji! 🥇 🥇 🥇 Byl to jeden z mnoha mých dluhů... :/

Reply:
Dobrý point